examples/SimpleTxtPlugin/SimpleTxtPlugin.csproj

```diff
@@ -5,12 +5,12 @@
     <Nullable>enable</Nullable>
     <EnableDynamicLoading>true</EnableDynamicLoading>
     <GeneratePluginPackageOnBuild>true</GeneratePluginPackageOnBuild>
-    <RootNamespace>Zeiss.FirstImportFormat</RootNamespace>
+    <RootNamespace>SimpleTxtPlugin</RootNamespace>
     <Authors>Carl Zeiss IMT GmbH</Authors>
     <Company>Carl Zeiss IMT GmbH</Company>
     <Product>Zeiss.FirstImportFormat</Product>
     <PackageId>Zeiss.FirstImportFormat</PackageId>
```

Comment:
Can you remove this property? That's only used for creating NuGet packags which is unrelated to import plugins.

Reply:
Oh, sure.